unity/Assets/Scripts/Unity/UnityTileHighlighter.cs

```diff
@@ -7,7 +7,7 @@ public class UnityTileHighlighter : MonoBehaviour
         //smaller -> slower
         private readonly float INVERSE_PERIOD_DURATION = 0.4f;
         private readonly float MAX_EMISSION_POWER = 0.2f;
-        private readonly Color GLOW_COLOR = Color.white;
+        public Color GLOW_COLOR = Color.white;
```

Comment:
Can these be constants? The way the field names are written suggests that. Even then, not really sure, if that's inline with the naming conventions we've set.

Reply:
ah, no, you're right. I didnt change the naming but should've as its not a constant anymore.